### PR TITLE
Remove references to module variables

### PR DESCRIPTION
**Pull request by spusateri**
This PR is meant to resolve unwanted behaviour with dynaconf loading rhsso settings for all tests that import the module, even those that do not explicitly use them. When the settings are stored as variables, they are loaded each time the module is imported, whether they are used in the test or not. To fix this unwanted behaviour, I replaced the reference to the variable with the value it was defined as in the original code.

### CODE REVIEW

File: robottelo/rhsso_utils.py [hunk 1]
```diff
@@ -50,16 +45,16 @@ def get_rhsso_client_id():
...
             settings.rhsso.host_url.replace('https://', 'http://'),
-            realm,
-            rhsso_user,
-            rhsso_password,
+            settings.rhsso.realm,
+            settings.rhsso.rhsso_user,
+            settings.rhsso.rhsso_password,
         ),
-        hostname=rhsso_host,
+        hostname=str(settings.rhsso.host_name),
```
Comment (mshriver) [inline robottelo/rhsso_utils.py:52]: These `str()` wrappers can be removed, its an artifact of the SettingsNodeWrapper behavior.

File: robottelo/rhsso_utils.py [hunk 2]
```diff
@@ -114,44 +109,44 @@ def create_new_rhsso_user(client_id, username=None):
...
     )
     return RHSSO_NEW_USER
 
 
 def update_rhsso_user(username, group_name=None):
     user_details = get_rhsso_user_details(username)
-    RHSSO_USER_UPDATE["realm"] = f"{realm}"
+    RHSSO_USER_UPDATE["settings.rhsso.realm"] = f"{settings.rhsso.realm}"
```
Comment (mshriver) [inline robottelo/rhsso_utils.py:131]: The string use of `"realm"` for the dict key shouldn't be changed.

Also, single quotes.

**Review by mshriver — CHANGES_REQUESTED** [review]
One required change to dict key reference

File: robottelo/rhsso_utils.py (hunk 2 above)
Comment (spusateri) [reply robottelo/rhsso_utils.py]: Added in latest commit

**Comment by spusateri** [comment]
Added requested changes in latest commit

**Comment by mshriver** [comment]
@spusateri check the GHA checks, you've got some pre-commit failures.

**Review by ColeHiggins2 — APPROVED** [review]
ACK, pending pre-commit fixes

File: robottelo/rhsso_utils.py [hunk 3]
```diff
@@ -179,7 +179,7 @@ def update_client_configuration(json_content):
     update_cmd = (
         f"{KEY_CLOAK_CLI} update clients/{client_id} -f update_client_info -s enabled=true --merge"
     )
-    run_command(cmd=update_cmd, hostname=settings.rhsso.host_name))
```
Comment (jaryn) [inline robottelo/rhsso_utils.py]: How comes this was not caught by some linter?

**Review by mshriver — APPROVED** [review]

**Pull request merged**